src/music/chord.py

```diff
@@ -1,12 +1,14 @@
 from . import base_sound
-
+from utils import constants
 
 class Chord(base_sound.BaseSound):
     def __init__(self, note=None, symbol=None, duration=None,
                  kind=None):
         super().__init__(note, symbol)
-        self.kind = kind
-        self.duration = duration
+        if kind is not None:
+            self.kind = kind
```

Comment:
Is it better that chord doesn't have property `kind` than it has property `kind` equal `None`?

Reply:
It's better when you don't know if there is any setter in the base class that will throw an exception when trying to set None

Reply:
ok